modules/wallet/seed.go

```diff
@@ -229,16 +187,60 @@ func (w *Wallet) LoadSeed(masterKey crypto.TwofishKey, seed modules.Seed) error
 	}
 	defer w.tg.Done()
 
+	// Because the recovery seed does not have a UID, duplication must be
+	// prevented by comparing with the list of decrypted seeds. This can only
+	// occur while the wallet is unlocked.
+	w.mu.RLock()
+	if !w.unlocked {
+		w.mu.RUnlock()
+		return modules.ErrLockedWallet
+	}
+	for _, wSeed := range append([]modules.Seed{w.primarySeed}, w.seeds...) {
+		if seed == wSeed {
+			w.mu.RUnlock()
+			return errKnownSeed
+		}
+	}
+	w.mu.RUnlock()
+
+	// scan blockchain to determine how many keys to generate for the seed
+	s := newSeedScanner(seed)
+	if err := s.scan(w.cs); err != nil {
+		return err
+	}
+	// Add 10% as a buffer because the seed may have addresses in the wild
+	// that have not appeared in the blockchain yet.
+	seedProgress := s.largestIndexSeen + 1
+	seedProgress += seedProgress / 10
```

Comment:
can you add a log here with both the largest seen and the final value for seedProgress?

I would like to check in my wallet what these values ended up being.